src/main/java/com/minecolonies/api/items/ModToolTypes.java

```diff
@@ -0,0 +1,283 @@
+import net.minecraftforge.common.ToolAction;
+import net.minecraftforge.common.ToolActions;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Supplier;
+
+/**
+ * Class used for storing and registering any ToolTypes.
+ */
+public class ModToolTypes
+{
+    /**
+     * The Forge registry for tool types. Do not directly add to this registry. Prefer using the
+     * register() function provided here.
+     */
+    private final static DeferredRegister<ToolTypeEntry> DEFERRED_REGISTER = DeferredRegister.create(new ResourceLocation(Constants.MOD_ID, "tooltypes"), Constants.MOD_ID);
+
+    public static String NONE_ID            = "none";
+    public static String PICKAXE_ID         = "pickaxe";
+    public static String SHOVEL_ID          = "shovel";
+    public static String AXE_ID             = "axe";
+    public static String HOE_ID             = "hoe";
+    public static String SWORD_ID           = "sword";
+    public static String BOW_ID             = "bow";
+    public static String FISHING_ROD_ID     = "rod";
+    public static String SHEARS_ID          = "shears";
+    public static String SHIELD_ID          = "shield";
+    public static String HELMET_ID          = "helmet";
+    public static String LEGGINGS_ID        = "leggings";
+    public static String CHESTPLATE_ID      = "chestplate";
+    public static String BOOTS_ID           = "boots";
+    public static String FLINT_AND_STEEL_ID = "flintandsteel";
+
+    public static RegistryObject<ToolTypeEntry> none;
+    public static RegistryObject<ToolTypeEntry> pickaxe;
+    public static RegistryObject<ToolTypeEntry> shovel;
+    public static RegistryObject<ToolTypeEntry> axe;
+    public static RegistryObject<ToolTypeEntry> hoe;
+    public static RegistryObject<ToolTypeEntry> sword;
+    public static RegistryObject<ToolTypeEntry> bow;
+    public static RegistryObject<ToolTypeEntry> fishing_rod;
+    public static RegistryObject<ToolTypeEntry> shears;
+    public static RegistryObject<ToolTypeEntry> shield;
+    public static RegistryObject<ToolTypeEntry> helmet;
+    public static RegistryObject<ToolTypeEntry> leggings;
+    public static RegistryObject<ToolTypeEntry> chestplate;
+    public static RegistryObject<ToolTypeEntry> boots;
+    public static RegistryObject<ToolTypeEntry> flint_and_steel;
+
+    /**
+     * The list of all tool types.
+     */
+    public static List<RegistryObject<ToolTypeEntry>> toolTypes = new ArrayList<>();
+    static
+    {
+        none = register(NONE_ID,
+          () -> new ToolTypeEntry.Builder().setName("")
+                  .setDisplayName(Component.translatable(ToolTranslationConstants.TOOL_TYPE_NONE))
+                  .setIsTool(itemStack -> true)
+                  .setToolLevel(itemStack -> itemStack == null ? 0 : 1)
```

Comment:
Would probably have this return `-1` by default, as tooltype none isn't really intended to be used to begin with.

Reply:
Done.